src/modules/land_detector/MulticopterLandDetector.h

```diff
@@ -97,35 +105,22 @@ class MulticopterLandDetector : public LandDetector
 	* @brief Handles for interesting parameters
 	**/
 	struct {
-		param_t maxClimbRate;
-		param_t maxVelocity;
-		param_t maxRotation;
 		param_t minThrottle;
 		param_t hoverThrottle;
 		param_t minManThrottle;
-		param_t freefall_acc_threshold;
-		param_t freefall_trigger_time;
-		param_t altitude_max;
 		param_t landSpeed;
-		param_t low_thrust_threshold;
 	} _paramHandle{};
 
 	struct {
-		float maxClimbRate;
-		float maxVelocity;
-		float maxRotation_rad_s;
 		float minThrottle;
 		float hoverThrottle;
 		float minManThrottle;
-		float freefall_acc_threshold;
-		float freefall_trigger_time;
-		float altitude_max;
 		float landSpeed;
-		float low_thrust_threshold;
 	} _params{};
 
 	uORB::Subscription _actuator_controls_sub{ORB_ID(actuator_controls_0)};
 	uORB::Subscription _battery_sub{ORB_ID(battery_status)};
+	uORB::Subscription _param_update_sub{ORB_ID(parameter_update)};
```

Comment:
Why do you need this? It is handled in the base class already.

Reply:
Thanks @bkueng!  Removed with upstream merge while resolving conflicts.